docs/ref/forms/fields.txt

```diff
@@ -299,6 +299,18 @@ as the rendered output.
 See the :ref:`format localization <format-localization>` documentation for
 more information.
 
+``disabled``
+~~~~~~~~~~~~~~~~
+
+.. attribute:: Field.disabled
+
+.. versionadded:: 1.9
+
+The ``disabled`` boolean argument, when set to ``True``, allows displaying a
+form field in a disabled way, that is its widget will have the ``disabled``
```

Comment:
... set to True, disables a form field using the disabled HTML attribute so that it won't be ...